### DISCUSSION TITLE
stderr: cisst_ros2_bridge when build with ROS2 humble

### DISCUSSION THREAD

**Discussion by sailboy1025** [jhu-dvrk/sawIntuitiveResearchKit#179] [Q&A] [answered]
I'm trying to build dVRK with ROS2 following [this](https://github.com/jhu-dvrk/sawIntuitiveResearchKit/wiki/BuildROS2), seems like a file is missing. Any suggestion would be helpful!
![1686035485874](https://github.com/jhu-dvrk/sawIntuitiveResearchKit/assets/124316691/ecd80524-c6f0-45ba-8f7c-7c21eeac378f)

Thanks,
Sai Jiang

**Comment by adeguet1** [comment, marked as answer]
My bad, I forgot to remove the `#include` for an old file.  It happened to work on my build tree because the old generated file was still there.  I just pushed a fix.  You should be able to get the latest code using `vcs pull` in your `src` folder.

https://github.com/jhu-cisst/cisst_ros2_bridge/commit/ee4f982612168dff4d2b69688ddfb0e96a498e34

**Reply by sailboy1025** [reply]
Thank you！
